src/routes.js

```diff
@@ -95,15 +69,16 @@ export default function getRoutes(
 			request.log(['info'], chalk.green(`Request received for ${request.url.href} (${requestLanguage})`));
 
 			const render$ = request.authorize()  // `authorize()` method is supplied by anonAuthPlugin
-				.flatMap(renderRequestMap[requestLanguage]);
+				.flatMap(renderRequestMap[requestLanguage])
+				.do(() => request.log(['info'], chalk.green('HTML response ready')));
 
 			render$.subscribe(
 				({ result, statusCode }) => {
 					// response is sent when this function returns (`nextTick`)
-					const response = reply(result).code(statusCode);
-					initTrackingCookie(request, response);
+					const response = reply(result)
+						.code(statusCode);
+					reply.track(response, 'session');
```

Comment:
only if there isn't a session cookie in the request https://github.com/meetup/meetup-web-platform/pull/37/files#diff-6357509bb7830b2c8e6b6183aa9ee069R21